docs/standard/datetime/toc.yml

```diff
@@ -39,3 +39,5 @@
     href: resolve-ambiguous-times.md
   - name: "How to: Let users resolve ambiguous times"
     href: let-users-resolve-ambiguous-times.md
+  - name: "How to: Let users resolve ambiguous times"
+    href: let-users-resolve-ambiguous-times.md
```

Comment:
This TOC entry appears to be a copy-paste of the one above; I guess, it was intended to be transformed into the link to the new article.